src/chapters/exceptions/throw.rst

```diff
@@ -0,0 +1,64 @@
+Throw
+=====
+
+In most programming languages when the compiler, or interpreter encounters code it doesn't know how to handle it has to notify the programmer that something has gone wrong. We call this act "throwing an exception". This is where the term throw comes from.
```

Comment:
"In most programming languages, when the compiler or interpreter encounters code it doesn't know how to handle, it throws an exception. This is how the compiler notifies the programmer that something has gone wrong."

The last sentence is unnecessary.

Reply:
fixed